docs/content/user-guide/en/monitoring/dashboard.md

```diff
@@ -35,66 +35,87 @@ You can change the path of the Dashboard by modifying this configuration option.
 
 This configuration option is used to configure the Dashboard front end to get the polling time of the status interface (/stats).
 
-* UseAuth 
+* AllowAnonymousExplicit
 
-> Default：false
+> Default: true
 
-Enable authentication on dashboard request.
+Explicitly allows anonymous access for the CAP dashboard API, passing AllowAnonymous to the ASP.NET Core global authorization filter.
 
-* DefaultAuthenticationScheme 
+* AuthorizationPolicy
 
-Default scheme used for authentication. If no scheme is set, the DefaultScheme set up in AddAuthentication will be used.
+> Default: null.
 
-* UseChallengeOnAuth
+Authorization policy for the Dashboard. Required if `AllowAnonymousExplicit` is false.
 
-> Default：false
+###  Custom Authentication
 
-Enable authentication challenge on dashboard request.
+From version 7.2.3, the CAP Dashboard leverages ASP.NET Core authentication mechanisms allowing extensibility through custom authorization policies and ASP.NET Core authentication and authorization middlewares to authorize Dashboard access. For more details of ASP.NET Core authentication internals, check [the official docs](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/?view=aspnetcore-8.0).
```

Comment:
@yang-xiaodong I was not sure if this would be the release version containing this change. If not, kindly let me know so that I can update the file! 🙏

Reply:
Since this is a breaking update so this PR will be merged into the next minor or major version, I will update the version description here, thanks